limitador-server/src/main.rs

```diff
@@ -462,6 +477,12 @@ fn create_config() -> (Configuration, String) {
                 .display_order(6)
                 .help("Sets the level of verbosity"),
         )
+        .arg(
+            Arg::with_name("validate")
```

Comment:
This will be documented as part of the limitador-server v1.0.0 crate, right?

Reply:
There is no such crate. But yes, there is task on-going to document _all_ these things. This will also be in the migration guide.